src/controllers/csi/gc/reconciler.go

```diff
@@ -75,11 +75,6 @@ func (gc *CSIGarbageCollector) Reconcile(ctx context.Context, request reconcile.
 		return defaultReconcileResult, err
 	}
 
-	if !isSafeToGC(ctx, gc.db, dynakubeList) {
```

Comment:
unnecessary as the GC no longer runs at the same time as the provisioner

Reply:
but why now?: it relied on the Version/Tag in the spec/status, so this logic would need update due to the Status changes, which is kinda useless work if its not needed in the first place